Doc/library/os.rst

```diff
@@ -911,6 +915,8 @@ as internal buffering of data.
    This function can support :ref:`paths relative to directory descriptors
    <dir_fd>` with the *dir_fd* parameter.
 
+   .. audit-event:: open "path mode flags"
```

Comment:
Is this ``open`` or ``os.open``?

Reply:
`open` is raised from a lot of places, regardless of how it's called. Distinguishing `os.open` from any other way a file is accessed just makes things harder for hooks.